.gitignore

```diff
@@ -17,3 +17,4 @@ config.h
 config.h.in
 mpdscribble
 .dirstamp
+objdir
```

Comment:
What's this?

Reply:
`objdir` is where I'm typically running `../configure && make -j ${SMP}`

Reply:
I don't think your personal choice of output directory needs to be part of the mpdscribble project, and it doesn't belong in this commit.